kinetic/distribution.yaml

```
@@ -4549,6 +4549,11 @@ repositories:
       url: https://github.com/ccny-ros-pkg/imu_tools.git
       version: kinetic
     status: developed
+  indoor_localization:
+    doc:
+      type: git
+      url: https://github.com/inomuh/indoor_localization.git
+      version: master  
```

This line has trailing whitespace which is failing the linter.

```suggestion
      version: master
```